amundsen_application/static/js/components/common/Bookmark/BookmarkIcon/styles.scss

```diff
@@ -5,7 +5,7 @@
   cursor: pointer;
   display: inline-block;
   height: 32px;
-  margin: -6px 0 -8px 4px;
+  margin-left: 4px;
```

Comment:
If this isn't a merge conflict and affects the search results UI, this should be done on master or we will have conflicting work.